api/serializers.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import serializers
+from .models import address, cityID, stateID
+
+class AddressGetSerializer(serializers.ModelSerializer):
+    #Added 2 additional fields which are derived from related 'cityID' and 'stateID' models
+    city_name = serializers.CharField(source='city.city_name', read_only=True)
+    state_name = serializers.CharField(source='state.state_name', read_only=True)
+    
+    class Meta:
+        model = address
+        fields = ('id', 'username', 'line1', 'line2', 'city_name', 
+                  'state_name', 'country', 'pin_code', 'phone_number', 
+                  'is_default')
+
+
+class AddressCreateSerializer(serializers.ModelSerializer):
+    # used for input during deserialization but won't be included 
+    # in the serialized output.
+    city_name = serializers.CharField(write_only=True)  
+    state_name = serializers.CharField(write_only=True)
+    # only be included in the serialized output  
```

Comment:
```suggestion

    # will only be included in the serialized output  
```

add new line to group related fields.